### PR TITLE
Remove index template from backend

### PR DESCRIPTION
**Pull request by CohenIdo**

### CODE REVIEW

**Comment by kfirpeled** [comment]
Hey @CohenIdo, please reference to the related PR on `integrations` repo

**Comment by CohenIdo** [comment]
[integrations repo PR](https://github.com/build-security/integrations/pull/3)

**Review by kfirpeled — APPROVED** [review]

File: x-pack/plugins/cloud_security_posture/common/constants.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@
  * 2.0.
  */
 
-export const CSP_KUBEBEAT_INDEX_PATTERN = 'logs-k8s_cis*';
+export const CSP_KUBEBEAT_INDEX_PATTERN = 'logs-k8s_cis.result*';
```
Comment (kfirpeled) [inline x-pack/plugins/cloud_security_posture/common/constants.ts:8]: is this change necessary?

**Review by ari-aviran — CHANGES_REQUESTED** [review]
Lets put this on hold for now until we merge our code to upstream Kibana, we need to have an easy setup, and since we don't have an integration working yet, lets keep it in Kibana server for now

**Comment by ari-aviran** [comment]
@CohenIdo Please rebase this to solve conflicts and we'll merge this in

**Review by ari-aviran — APPROVED** [review]

File: x-pack/plugins/cloud_security_posture/server/index_template/findings_mapping.ts [hunk 2]
```diff
@@ -1,184 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License
- * 2.0; you may not use this file except in compliance with the Elastic License
- * 2.0.
- */
-import type { MappingTypeMapping } from '@elastic/elasticsearch/lib/api/types';
-
-export const mapping: MappingTypeMapping = {
```
Comment (orouz) [inline x-pack/plugins/cloud_security_posture/server/index_template/findings_mapping.ts:9]: FYI the current findings needs this to do search/sort @kfirpeled

**Pull request merged**